### PR TITLE
Add commonised JSON to scrabble score problem

### PR DESCRIPTION
**Pull request by SwamWithTurtles**

### CODE REVIEW

File: scrabble-score.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "cases": [
+    {
+      "description": "Short input",
+      "input": "a",
+      "expected": 1,
+      "msg": "No, 'a' scores 1 point"
+    },
+    {
+      "description": "Uppercase Short input",
```
Comment (kytrinyx) [inline scrabble-score.json]: `s/Short/short` I think.

File: scrabble-score.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      "msg": "No, 'street' scores 6 point"
+    },
+    {
+      "description": "Medium word, with letters worth multiple points",
+      "input": "quirky",
+      "expected": 22,
+      "msg": "No, 'quirky' scores 22 point"
+    },
+    {
+      "description": "Upper case long word",
```
Comment (kytrinyx) [inline scrabble-score.json]: s/Upper case/uppercase/

File: scrabble-score.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      "input": "MULTIBILLIONAIRE",
```
Comment (kytrinyx) [inline scrabble-score.json]: We shouldn't include this word, since it couldn't ever occur in a real game.

File: scrabble-score.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      "expected": 20,
+      "msg": "No, 'MULTIBILLIONAIRE' scores 20 point"
+    },
+    {
+      "description": "Mixed Case long word",
+      "input": "OxyphenButazone",
```
Comment (kytrinyx) [inline scrabble-score.json]: We added this one because we had to get rid of multibillionaire (I had no idea. Thankfully there are scrabble fiends out there who know stuff like this).

File: scrabble-score.json [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+      "msg": "No, 'quirky' scores 22 point"
+    },
+    {
+      "description": "Upper case long word",
+      "input": "MULTIBILLIONAIRE",
+      "expected": 20,
+      "msg": "No, 'MULTIBILLIONAIRE' scores 20 point"
+    },
+    {
+      "description": "Mixed Case long word",
```
Comment (kytrinyx) [inline scrabble-score.json]: `s/Case/case/`

File: scrabble-score.json [hunk 6]
```diff
@@ -0,0 +1,64 @@
+{
+  "cases": [
+    {
+      "description": "Short input",
+      "input": "a",
+      "expected": 1,
+      "msg": "No, 'a' scores 1 point"
```
Comment (kytrinyx) [inline scrabble-score.json]: Let's ditch the `msg` in this exercise; it's not adding any information beyond the expected.

We needed it for one exercise where the expected was just a boolean, and the assertions ended up being confusing.

**Comment by kytrinyx** [comment]
I have a few small comments, but nothing major. Thanks!

**Comment by kytrinyx** [comment]
@SwamWithTurtles Any chance you'll have time to make the changes on this one? I'd love to merge it.

**Comment by IanWhitney** [comment]
I'm working on implementing this in Rust, and I've followed this proposed PR. But I added one test for the edge case of Non-English characters:

https://github.com/exercism/xrust/pull/125/files#diff-26a63b91a0b305a1fd995e76a5ef5ad2R54

Though this is particularly useful in Rust (or other languages with Optional types), I can it being a useful test in other languages as well.

**Comment by kytrinyx** [comment]
Superseded by #255 - thanks for your work on this one @SwamWithTurtles!
